From cebab8517d030ae193e9b47741211a576f795c0b Mon Sep 17 00:00:00 2001
From: Yann Dirson <yann.dirson@vates.fr>
Date: Wed, 23 Aug 2023 11:00:40 +0200
Subject: [PATCH] Look for firmware in /lib/firmware/override/

This -alt driver installs in /lib/modules/*/override/ to coexit with
the non-alt package, its firmware files need to be installed in a
different directory as well, and loaded from there.

This patch is only necessary until the kernel gets
/lib/firmware/override/ in its firmware search path.
---
 src/ice_main.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/ice_main.c b/src/ice_main.c
index ab5e57c..e8f409a 100644
--- a/src/ice_main.c
+++ b/src/ice_main.c
@@ -45,7 +45,7 @@ static const char ice_copyright[] = "Copyright (C) 2018-2021, Intel Corporation.
 #if UTS_UBUNTU_RELEASE_ABI
 #define ICE_DDP_PKG_PATH	"updates/intel/ice/ddp/"
 #else /* UTS_UBUNTU_RELEASE_ABI */
-#define ICE_DDP_PKG_PATH	"intel/ice/ddp/"
+#define ICE_DDP_PKG_PATH	"override/intel/ice/ddp/"
 #endif /* UTS_UBUNTU_RELEASE_ABI */
 #define ICE_DDP_PKG_FILE	ICE_DDP_PKG_PATH "ice.pkg"
 
-- 
2.30.2

